picard/tagger.py

```diff
@@ -322,8 +344,21 @@ def pipe_server(self):
         while self.pipe_handler.pipe_running:
             messages = [x for x in self.pipe_handler.read_from_pipe() if x not in IGNORED]
             if messages:
-                self.add_paths(messages)
-                self.bring_tagger_front()
+                self.load_to_picard(messages)
+
+    def load_to_picard(self, items):
+        parsed_items = ParseItemsToLoad(items)
+
+        if parsed_items.files:
+            self.add_paths(parsed_items.files)
+            self.bring_tagger_front()
+        if parsed_items.urls or parsed_items.mbids:
+            file_lookup = self.get_file_lookup()
+            for mbid in parsed_items.mbids:
+                thread.to_main(file_lookup.mbid_lookup, mbid, None, None, False)
+            for url in parsed_items.urls:
+                thread.to_main(file_lookup.mbid_lookup, url, None, None, False)
```

Comment:
Minor point, but since we're dealing with sets here I think this could be shortened a bit by using the combined sets as the iterator (unless you want to keep the duplicated code for clarity).  I haven't tested this, but I think the following should work:

```suggestion
            for item in parsed_items.mbids.union(parsed_items.urls):
                thread.to_main(file_lookup.mbid_lookup, item, None, None, False)
```

Reply:
I tend to prefer the set union using the `|` operator which is easier to read to me, dunno which one you prefer:

```suggestion
            for item in parsed_items.mbids | parsed_items.urls:
                thread.to_main(file_lookup.mbid_lookup, item, None, None, False)
```

Reply:
You're right.  That is easier to read.  I actually prefer it the way you have it written.

Reply:
I'm okay with that